layouts/default.html

```diff
@@ -19,7 +19,9 @@
     <div id="header-wrapper">
       <div id="header">
         <div>
-          <a class="logo" href="/"><img src="/images/logo.png" alt="SeeClickFix API" /></a>
+          <a class="logo" href="/">
+            <img src="/images/seeclickfix-311-crm-logo.png" height="33px" alt="CivicPlus SeeClickFix 311 CRM" />
```

Comment:
@CourtneyWill I see the actual image size is 440w x 108h.  Was that an intentional asset size choice, for the 33px rendered height?

Reply:
@zbeat I don't see an option to export a custom dimension. There is only the preset option [1x, 1.5x, 2x, 3x.....].
I exported the file and set the size in HTML.